kofta/public/locales/ta/translation.json

```diff
@@ -1,250 +1,243 @@
-    },
-    "pages": {
-      "_comment": "Respective Page UI Internationalization Strings",
-      "banUser": {
-        "ban": "தடை செய்"
-      },
-      "followingOnlineList": {
-        "listHeader": "ஒரு தனியார் அறையில் இல்லாத, நீங்கள் பின்பற்றும் பயனர்களின்.",
-        "currentRoom": "தற்போது இருப்பவர்கள்:",
-        "startPrivateRoom": "அவர்களுடன் ஒரு தனியார் அறையைத் தொடங்குங்கள்"
-      },
-      "followList": {
-        "followHim": "பின்தொடரவும்",
-        "followingHim": "இவர்களை பின்தொடர்கிறீர்கள்"
-      },
-      "home": {
-        "createRoom": "அறையை உருவாக்குங்கள்",
-        "refresh": "புதுப்பிக்கவும்"
-      },
-      "inviteList": {
-        "roomGone": "அறை இல்லை, திரும்பிச் செல்லுங்கள்",
-        "shareRoomLink": "அறைக்கு இணைப்பைப் பகிரவும்",
-        "inviteFollowers": "ஆன்லைனில் இருக்கும் உங்களைப் பின்தொடர்பவர்களை அழைக்கலாம்:",
-        "whenFollowersOnline": "உங்களைப் பின்தொடர்பவர்கள் ஆன்லைனில் இருக்கும்போது, இங்கே காண்பிக்கப்படுவார்கள்."
-      },
-      "login": {
-        "headerText": "உங்கள் குரலை நிலாவுக்கு பேக் செய்கிறோம் 🚀",
-        "featureText_1": "Dark Theme",
-        "featureText_2": "Open Sign-Ups",
-        "featureText_3": "Cross-Platform Support",
-        "featureText_4": "Open Source",
-        "featureText_5": "எழுத்து அரட்டை",
-        "featureText_6": "இது Doge ஆல் பவர் செய்யப்பட்டது  ",
-        "loginGithub": "GitHub மூலமாக உள்நுழை",
-        "loginTwitter": "Twitter மூலமாக உள்நுழை",
-        "createTestUser": "சோதனை பயனரை உருவாக்கவும்"
-      },
-      "myProfile": {
-        "logout": "லாக் அவுட் செய்",
-        "probablyLoading": "பதிவேறுகிறது என்று நினைக்கிறன்...",
-        "voiceSettings": "குரல் அமைப்பிற்குச் செல்லவும்",
-        "soundSettings": "ஒலி அமைப்பிற்குச் செல்லவும்",
-        "deleteAccount": "கணக்கை நீக்கு"
-      },
-      "notFound": {
-        "whoopsError": "அய்யகோ! இந்த பக்கத்தை காணவில்லை",
-        "goHomeMessage": "கவலை படாதே, ",
-        "goHomeLinkText": "Home'இற்கு செல் "
-      },
-      "room": {
-        "speakers": "பேசுகின்றவர்கள்",
-        "requestingToSpeak": "பேசுவதற்கு கோரிக்கை வைக்கின்றேன்",
-        "listeners": "கவனிக்கின்றவர்கள்"
-      },
-      "searchUser": {
-        "search": "தேடு..."
-      },
-      "soundEffectSettings": {
-        "header": "ஒலிகள்"
-      },
-      "viewUser": {
-        "editProfile": "சுயவிவரத்தைத் திருத்து",
-        "followsYou": "உங்களைப் பின்தொடர்கிறார்",
-        "followers": "உங்களைப் பின்தொடர்பவர்கள்",
-        "following": "நீங்கள் பின்தொடர்பவர்கள் ",
-        "followHim": "இவரை பின்தொடர்",
-        "followingHim": "இவரை பிந்தொடர்கிறீர்கள்"
-      },
-      "voiceSettings": {
-        "header": "குரல் அமைப்பு",
-        "mic": "ஒலிவாங்கி:",
-        "permissionError": "ஒலிவாங்கிகள் எதுவும் கிடைக்கவில்லை, நீங்கள் எதுவும் செருகவில்லை அல்லது இந்த வலைத்தளத்திற்கு அனுமதி வழங்கவில்லை.",
-        "refresh": "ஒலிவாங்கி பட்டியலைப் புதுப்பிக்கவும்",
-        "volume": "ஒலி அளவு:"
-      }
```

Comment:
What's the large diff in this file about? Is it just indenting?

Reply:
It looks like some indenting auto-formatting done after running the `npm run i18` command

Reply:
Got ya